vMenu/MainMenu.cs

```diff
@@ -56,11 +59,18 @@ public class MainMenu : BaseScript
         public static bool DebugMode = GetResourceMetadata(GetCurrentResourceName(), "client_debug_mode", 0) == "true" ? true : false;
         public static bool EnableExperimentalFeatures = /*true;*/ (GetResourceMetadata(GetCurrentResourceName(), "experimental_features_enabled", 0) ?? "0") == "1";
         public static string Version { get { return GetResourceMetadata(GetCurrentResourceName(), "version", 0); } }
+        
+        public static Dictionary<string, Hashtable> LanguageData = new Dictionary<string, Hashtable>();
```

Comment:
Please don't store these things in the MainMenu class, but instead in the LanguageManager class.